pkg/dyninst/testdata/decoded/sample/testEmptyStruct.json

```diff
@@ -26,7 +26,7 @@
             "arguments": {
               "e": {
                 "type": "main.emptyStruct",
-                "fields": {}
+                "notCapturedReason": "unavailable"
```

Comment:
I think the output before was better. We want to do something different for zero-byte expressions. There's literally nothing to capture. What we want to do, I think, is consult the type's byte size even before looking at the presence bitset. If it's zero-bytes, we always say we have it.

Reply:
Ah - yea. Good catch, thanks.